lightly_studio/src/lightly_studio/api/routes/api/frame.py

```diff
@@ -202,8 +204,12 @@ def _build_sample_view(sample: SampleTable) -> SampleView:
         created_at=sample.created_at,
         updated_at=sample.updated_at,
         tags=sample.tags,
-        metadata_dict=sample.metadata_dict,
-        captions=sample.captions,
+        metadata_dict=SampleMetadataView.model_validate(sample.metadata_dict)
```

Comment:
Nit: I would put the resolution of what metadata_dict is before the function call to keep the arguments more readable

Reply:
Do you mean something like this?

```
metadata_dict = sample.metadata_dict

...

metadata_dict=SampleMetadataView.model_validate(metadata_dict)

```